spec/rubocop/cop/style/ternary_parentheses_spec.rb

```diff
@@ -266,6 +266,9 @@
       it_behaves_like 'code with offense',
                       'foo = bar or (baz) ? a : b',
                       'foo = bar or baz ? a : b'
+
+      it_behaves_like 'code without offense',
+                      'foo = bar&.baz ? a : b'
```

Comment:
It may be consistent with other test cases if a test case is written as follows.

```diff
diff --git a/spec/rubocop/cop/style/ternary_parentheses_spec.rb b/spec/rubocop/cop/style/ternary_parentheses_spec.rb
index f0438b723..9ef7e6254 100644
--- a/spec/rubocop/cop/style/ternary_parentheses_spec.rb
+++ b/spec/rubocop/cop/style/ternary_parentheses_spec.rb
@@ -267,7 +267,8 @@ RSpec.describe RuboCop::Cop::Style::TernaryParentheses, :config do
                       'foo = bar or (baz) ? a : b',
                       'foo = bar or baz ? a : b'

-      it_behaves_like 'code without offense',
+      it_behaves_like 'code with offense',
+                      'foo = (bar&.baz) ? a : b',
                       'foo = bar&.baz ? a : b'
     end
```

Reply:
Hi @koic, I have force-pushed the suggested change